src/Offer/CommandHandlers/DeleteDescriptionHandler.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Offer\CommandHandlers;
+
+use Broadway\CommandHandling\CommandHandler;
+use Broadway\EventStore\EventStreamNotFoundException;
+use CultuurNet\UDB3\Offer\Commands\DeleteDescription;
+use CultuurNet\UDB3\Offer\OfferRepository;
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\NullLogger;
+
+final class DeleteDescriptionHandler implements CommandHandler
+{
+    use LoggerAwareTrait;
```

Comment:
Nice to add logging but it is still required to call `setLogger` inside the bootstrapping of the `DeleteDescriptionHandler` otherwise the `NullLogger` is used.

Reply:
is $handler->setLogger(LoggerFactory::create($container, LoggerName::forWeb())); oke?

Reply:
Yes, that should do the trick.